CHANGES.markdown

```diff
@@ -1,3 +1,9 @@
+0.5.1 (3 Aug 2016)
```

Comment:
This should just be folded into the 0.5 changes below, I am not planning to make two separate 0.5 and 0.5.1 releases.